tests/scripts/setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+TUTORIAL_REPOSITORY=https://github.com/ezsystems/ezplatform-ee-beginner-tutorial
+STEP1_BRANCH=v2-step1
+STEP4_BRANCH=master
+TUTORIAL_DATA_DIRECTORY_1=tutorial_data_1
+TUTORIAL_DATA_DIRECTORY_2=tutorial_data_2
+IMAGES_SOURCE=vendor/ezsystems/developer-documentation/docs/tutorials/enterprise_beginner/img/photos.zip
+IMAGES_DESTINATION=./images
+
+# clone tutorial data
+git clone $TUTORIAL_REPOSITORY --depth=1 -b $STEP1_BRANCH $TUTORIAL_DATA_DIRECTORY_1
+git clone $TUTORIAL_REPOSITORY --depth=1 -b $STEP4_BRANCH $TUTORIAL_DATA_DIRECTORY_2
+
+# add suite to Behat
+echo '    - vendor/ezsystems/developer-documentation/tests/behat_suites.yml' >> behat.yml.dist
+
+#copy images
+mkdir $IMAGES_DESTINATION
+unzip $IMAGES_SOURCE -d $IMAGES_DESTINATION
+
+# copy templates, styles and configuration files
```

Comment:
This part adds all the files mentioned in https://doc.ezplatform.com/en/2.3/tutorials/enterprise_beginner/1_get_a_starter_website/#add-template-configuration-and-style-files

I didn't want to do all of this in PHP, it's much quicker this way.